runtime/parsing.js

```diff
@@ -147,7 +147,7 @@ function caml_parse_engine(tables, env, cmd, arg) {
           break;
         }
         res = READ_TOKEN;
-        break exit;
+        break the_loop;
```

Comment:
```suggestion
        break;
```

Reply:
You want to exit the loop and not just terminate the current switch statement.

Reply:
Indeed